src/_cffi_src/openssl/x509_vfy.py

```diff
@@ -200,6 +200,9 @@
                               size_t);
 int X509_VERIFY_PARAM_set1_ip_asc(X509_VERIFY_PARAM *, const char *);
 
+int sk_X509_OBJECT_num(Cryptography_STACK_OF_X509_OBJECT *);
+X509_OBJECT *sk_X509_OBJECT_value(Cryptography_STACK_OF_X509_OBJECT *, int);
+X509_VERIFY_PARAM * X509_STORE_get0_param(X509_STORE *);
```

Comment:
Style nit, this should be `X509_VERIFY_PARAM *X509_STORE_get0_param(X509_STORE *);` (space removed before function name)